src/App.jsx

```diff
@@ -12,11 +15,15 @@ console.log(`BACKEND_URL: ${BACKEND_URL}`);
 
 export default function App() {
   return (
-    <div className="container">
-      <div className="row">
-        <h1 className="page-title">Wow Shopping!</h1>
-        <h3> Empty Front End for Eric, Zack and Graham </h3>
-      </div>
-    </div>
+    <>
```

Comment:
may not need react fragments as it unnecessary since <BrowserRoute> is already wrapping the div